src/expr/src/linear.rs

```diff
@@ -1530,10 +1531,33 @@ pub mod plan {
                 }
             }
 
+            #[derive(Debug, Clone, Copy, Eq, PartialEq, Ord, PartialOrd)]
+            enum Bound {
+                TooLow,
+                InRange(u64),
+                TooBig,
```

Comment:
Names are inconsistent; should either be `TooLow` and `TooHigh` or `TooSmall` and `TooBig`.

Reply:
Only leaving such a minor nit because I know you care about consistency, @benesch ! Take it or leave it.

Reply:
Hah! Thanks, fixed.